config/runtime/tests/fluster-chromeos.jinja2

```diff
@@ -0,0 +1,61 @@
+{%- include 'chromeos/base.jinja2' %}
+
+{%- if testsuite -%}
+    {% set testsuite_arg = "-ts " + testsuite %}
+{%- endif -%}
+{%- if decoders -%}
+    {%- set decoders_arg = "-d " + decoders|join(" ") -%}
+{%- endif -%}
+{%- if videodec_timeout -%}
+    {% set videodec_timeout_arg = "-t " + videodec_timeout|string %}
+{%- endif -%}
+{%- if videodec_parallel_jobs -%}
+    {% set videodec_parallel_jobs_arg = "-j " + videodec_parallel_jobs|string %}
+{%- endif -%}
+
+- test:
+    namespace: chromeos
+    timeout:
+      minutes: {{ job_timeout }}
+    docker:
+      image: kernelci/cros-tast
+      wait:
+        device: true
+    results:
+      location: /home/cros/lava
+    definitions:
+    - from: inline
+      name: {{ node.name }}
+      path: inline/{{ node.name }}.yaml
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: {{ node.name }}
+        run:
+          steps:
+            - lava-test-set start setup
+            - for i in $(seq 1 60); do ping -c 1 -w 1 $(lava-target-ip) && break || sleep 1; done
+            - ping -c 1 -w 1 $(lava-target-ip) || lava-test-raise "cros-device-unreachable"
+            - lava-test-set stop setup
+            - >-
+              ./ssh_retry.sh
+              -o StrictHostKeyChecking=no
+              -o UserKnownHostsFile=/dev/null
+              -i /home/cros/.ssh/id_rsa
+              root@$(lava-target-ip)
+              python3 /usr/bin/fluster_parser.py {{ testsuite_arg }} {{ decoders_arg }} 
+                            {{ videodec_timeout_arg }} {{ videodec_parallel_jobs_arg }}
+            - >-
+              ./ssh_retry.sh
+              -o StrictHostKeyChecking=no
+              -o UserKnownHostsFile=/dev/null
+              -i /home/cros/.ssh/id_rsa
+              root@$(lava-target-ip)
+              python3 /usr/bin/fluster_parser.py --results
```

Comment:
This must be executed *after* poweroff (which implies that the test results should be available locally, and fluster_parser as well): if the test fails, then fluster_parser will error out, ending the LAVA job immediately, resulting in a hard shutdown.

Reply:
Thank you @a-wai . See https://github.com/kernelci/kernelci-core/pull/2617/commits/8312746098685b34342f98b66e5316051e07efe2